Orange/widgets/data/owfile.py

```diff
@@ -92,6 +93,11 @@ class Outputs:
         match_values=PerfectDomainContextHandler.MATCH_VALUES_ALL
     )
 
+    # pylint seems to want declarations separated from definitions?!
```

Comment:
I think this comment is not needed and should be removed.

Reply:
I'd keep it. It's not a complaint about pylint, it's here to explain why these lines are separated. Otherwise somebody could merge `recent_urls: List[str]` and `recent_urls = Setting([])` into `recent_urls: List[str] = Setting([])`, which is a legal way to do it, but pylint doesn't recognize it.

I removed `?!` though.